README.md

```diff
@@ -1 +1,51 @@
-# Portifolio_popup
\ No newline at end of file
+# Portifolio_popup
+
+
+This project is about creating a mobile view of my personal portifolio
+
+## Built with
+
+-HTML
+-CSS
+
+## setup to access the project
+
+> To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+-Github Account
+-code editor
+-Git
+
+### Setup
+
+(go to the terminal in the code editor and put the following git commands)
+-git init
+-git clone https://github.com/Kaweesi-Matia/mobile_website.git
+-cd portifolio-mobile
+-open index.html in the editor/browser to view the project contents
+-open style.css to see the CSS styles used in the project.
+-enjoy
+
+## Author
+
+
+### :MAN: Kaweesi Matia
+### Collaborator: Mudasir Sherwani
+
+-GitHub: @Kaweesi-Matia
+-Twitter:@kaweesimatia
+-LinkedIn: Kaweesi Matia
+
+### :handshake: Contributing
+
+Contributions,code reviews and feature requests are all welcome
+
+### Show your support
+
+Give a star if you liked this project
+
+### :memo: License
+
+This project is MIT licensed.
```

Comment:
- [ ] Please add a link to your license in your readme and also add the name and info of the people who are contributed to this project in author section to make your readme more professional :smile: